docs/performance.md

```diff
@@ -1,7 +1,543 @@
+`CVMFS_HTTP_PROXY` [client configuration
+settings](https://cvmfs.readthedocs.io/en/stable/apx-parameters.html#client-parameters).
+
+##### Cold cache
+
+Let's start with the worst-case [cold cache](#cold_cache) scenario, in which CernVM-FS needs to
+download all necessary files to the client system from a (local or remote) server before TensorFlow can be started.
+
+<p align="center">
+<img src="../img/perf/tensorflow-startup-cvmfs-cold.png" alt="Start-up performance of TensorFlow: CernVM-FS (cold
+caches)" width="90%"/></br>
+</p>
+
+We observe very reasonable cold start-up times when using an in-network private proxy server (~8.6 seconds)
+or private Stratum 1 replica server (~11.0 seconds), significantly better than what we observed for GPFS
+and Lustre (both of which are accessible from cluster worker nodes via a dedicated high-speed network).
+
+When CernVM-FS needs to download the necessary files from a remote public Stratum 1 server we of course
+observe dramatically longer start-up times, from ~62 seconds when using the relatively close one (within Europe), to
+about 6 *minutes* when having to go across the Atlantic.
+
+This highlights the need for a private proxy or Stratum 1 replica server for a production-quality CernVM-FS setup.
+
+##### Hot + warm cache
+
+In the [hot cache](#hot_cache) and [warm cache](#warm_cache) scenarios, in which all necessary files are
+already available locally on the client system either in-memory or on the local disk,
+we observe much better start-up times, as expected:
+
+<p align="center">
+<img src="../img/perf/tensorflow-startup-cvmfs-hot-warm.png" alt="Start-up performance of TensorFlow: CernVM-FS (hot + warm cache)" width="100%"/></br>
+</p>
+
+With a hot kernel file system cache, we see an average start-up time for TensorFlow of ~2.25 seconds,
+which suggests there is a ~7% overhead introduced by CernVM-FS. Note that this is still significantly
+better than what we observed for [NFS](#nfs), [Lustre](#lustre), and [GPFS](#gpfs).
+
+In the [warm cache](#warm_cache) scenarios where CernVM-FS serves all required files from its client cache
+on a local SSD disk we see start-up times that are on average ~3.9-4.3 seconds, depending on the network
+latency to the remote server used in the CernVM-FS client configuration (and interestingly,
+with a larger variance when using a local proxy or Stratum 1, which may be because of other activity
+on the local cluster network). We did not try to optimize these warm start-up
+times by changing the CernVM-FS configuration settings, but some options are already supported by CernVM-FS
+that could reduce the impact of network latency considerably.
+
+When using a RAM disk as location for the CernVM-FS client cache in combination with a private proxy server,
+the start-up time in the [warm cache](#warm_cache) scenario was reduced to ~2.56 seconds,
+and a lot less variation was observed (although that could be coincidental).
+
```

Comment:
do you want to add one final graphics that is an overview between all the different storage/FSs tested? (maybe just selected values)